### PR TITLE
add example: how to stub console.log

### PR DESCRIPTION
**Pull request by gabbersepp**
This shows how to stub console.log
As mentioned in #11 I should add this in the faq, too.
I will add this in the next days

added faq: https://github.com/cypress-io/cypress-documentation/pull/2169

### CODE REVIEW

**Comment by jennifer-shehane** [comment]
Taking a look at this now.

**Comment by bahmutov** [comment]
@gabbersepp can you accept my pull request into your branch first? https://github.com/gabbersepp/cypress-example-recipes/pull/1

File: snapshots.js [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (bahmutov) [inline snapshots.js]: what's this file?

**Comment by gabbersepp** [comment]
> @gabbersepp can you accept my pull request into your branch first? [gabbersepp#1](https://github.com/gabbersepp/cypress-example-recipes/pull/1)

Should be merged now. Hopefully everything was merged correctly. This is one of my first contributions, so these processes are very new to me.

File: snapshots.js (hunk 1 above)
Comment (gabbersepp) [reply snapshots.js]: Well good question. I have copied the example from the window spy folder and adjusted for my use case. If those files are irrelevant, they should be removed from that example, too. What do you think?

File: README.md [hunk 2]
```diff
@@ -72,6 +72,7 @@ Recipe | Description
 [Stubbing `window.fetch`](./examples/stubbing-spying__window-fetch) | Use `cy.stub()` to control fetch requests
 [Stubbing methods called on `window`](./examples/stubbing-spying__window) | Use `cy.stub()` for methods called on `window`
 [Stubbing Google Analytics](./examples/stubbing-spying__google-analytics) | Use `cy.stub()` to test Google Analytics calls
+[Stubbing methods called on `console`](./examples/stubbing-spying__console) | Use `cy.stub()` for methods called on `console`
```
Comment (bahmutov) [inline README.md]:
```suggestion
[Spying and stubbing methods on `console` object](./examples/stubbing-spying__console) | Use `cy.spy()` and `cy.stub()` on `console.log`
```

File: circle.yml [hunk 3]
```diff
@@ -148,6 +148,10 @@ jobs:
     <<: *defaults
   stubbing-spying__google-analytics:
     <<: *defaults
+  stubbing-spying__window:
```
Comment (bahmutov) [inline circle.yml]: nice catch - this recipe was missing from the build!

File: snapshots.js (hunk 1 above)
Comment (bahmutov) [reply snapshots.js]: yeah remove it from this file and from window recipe

Comment (gabbersepp) [reply snapshots.js]: I have deleted this file. No idea how this file was created. I will open an issue to cleanup the existing examples. Maybe there are more files that are not necessary.

Comment (gabbersepp) [reply snapshots.js]: https://github.com/cypress-io/cypress-example-recipes/issues/348

**Pull request merged**